packages/utilities/src/CliUtils.ts

```diff
@@ -143,9 +143,20 @@ export class CliUtils {
             } else if (profile != null) {
                 // For each option - extract the value if that exact property exists
                 options.forEach((opt) => {
-
                     // Get the camel an kebab case
-                    const cases = CliUtils.getOptionFormat(opt.name);
+                    let cases;
+                    if (profile[opt.name] == null) {
+                        // Use aliases for backwards compatibility
+                        // Search for first alias available in the profile
+                        if ("aliases" in opt) {
```

Comment:
Should these nested `if` statements be combined into a single one?

Reply:
combined both for now 😋